src/pngtocss.c

```diff
@@ -521,18 +521,20 @@ static int process_file(const char *fname)
 
 int main(int argc, char **argv)
 {
-	int i=1;
 	int result = 1;
 
 	if(argc == 1) {
 		version_info();
 		usage_info();
-		return result;
+		return EXIT_SUCCESS;
 	}
 
-	for(i=1; i<argc; i++)
+	for(int i=1; i<argc; i++)
```

Comment:
Okay, I have made the correction. I don't see the purpose of initializing `i` to 1 and then assigning `i` to 1 later (it was already 1?) so I made the adjustment. I'm not sure if this also goes against the ANSI standard as well or if there was some other reason behind this, please advise.